bin/setup_stack

```diff
@@ -100,12 +106,20 @@ sudo modprobe bonding
 bin/run_faucet $local corp 6683
 
 for index in 1; do
-    controller=forch-faucet-$index
+    if [ -n "$fot" ]; then
+        controller=forch-controller-$index
+    else
+        controller=forch-faucet-$index
+    fi
     mkdir -p inst/$controller/
     cp -r topo/$mode/forch inst/$controller/
     cp -r topo/$mode/faucet inst/$controller/
     docker kill $controller || true
-    bin/run_faucet $local $index $((6000 + index)) $((8000 + index))
+    if [ -n "$fot" ]; then
+        bin/run_controller $local $index $((6000 + index)) $((8000 + index))
+    else
+        bin/run_faucet $local $index $((6000 + index)) $((8000 + index))
+    fi
```

Comment:
gauge should be with run_faucet, so it's either { controller } or  { faucet, gauge } not { controller, gauge } and { faucet, gauge }

Reply:
Added gauge to container